src/java/org/apache/cassandra/tools/nodetool/SetMinimumKeyspaceRF.java

```diff
@@ -18,19 +18,31 @@
 
 package org.apache.cassandra.tools.nodetool;
 
+import java.util.ArrayList;
+import java.util.List;
+
 import io.airlift.airline.Arguments;
 import io.airlift.airline.Command;
 import org.apache.cassandra.tools.NodeProbe;
 import org.apache.cassandra.tools.NodeTool;
 
+import static java.lang.Boolean.parseBoolean;
+import static java.lang.Integer.parseInt;
+
 @Command(name = "setminimumrf", description = "Sets minimum keyspace replication factor.")
 public class SetMinimumKeyspaceRF extends NodeTool.NodeToolCmd
 {
-    @Arguments(title = "minimum_rf", usage = "<value>", description = "Minimum replication factor", required = true)
-    private Integer minimumRF = null;
+    @Arguments(title = "<setWarn> <value>", usage = "<setWarn> <value>", description = "warn flag and minimum keyspace rf value", required = true)
```

Comment:
In the initial guardrails patch we decided to use a single JMX setter for both warn and fail values (see [here](https://github.com/apache/cassandra/pull/1321#discussion_r760697524)), probably we should do the same here and require both values instead of a boolean.